### DISCUSSION TITLE
Soft text in Black & White

### DISCUSSION THREAD

**Discussion by Cix75** [trufanov-nok/scantailor-universal#104] [Ideas]
Hi, this is my second question... :p 
I will have to work on a lot of Magazines in color and a lot of books with pictures.
Just I noted that for the Text, using BW settings I really lost a lot of quality details of the image.
Using instead Grey/Color I have a bit more quality but I will not have a good white background sheet, it is always dirty compared to the BW one.
Using MIX, instead, I have the same result of the BW for the Text with color images good instead.
so the problem is almost on the text. There is a tool or some settings I can configure to have a result like the one in this picture, please?
![compare BW text](https://user-images.githubusercontent.com/63376261/149945068-e5e1d7cb-3b88-4f4a-a116-2f569d111237.png)

Here you can see a comparison image between the ScanDelay result and a clean using photoshop, I would like to keep a more softly text and not only a strong B&W, is there a way to obtain this result using ScanDelay maybe?
something like bilinear filtering on the B&W test pages. 
To obtain a similar result, usually, need to work pages in ScanTailor at 600dpi and then resize them in photoshop at 400dpi using bilinear soft filtering for each page.
So to have this possibility directly as an option in ScanTailor could really help me to save time, please :)

thanks for your help and effort :) W ScanTailor Universal!

**Comment by trufanov-nok** [comment]
Have you tried a *so called* Foreground layer option in Mixed mode? In this mode STU will binarize the image internally and instead of rendering bw result to output it will use the binarized image as a mask to separate the background from foreground. Background is filled with a white color while foreground is left as is (not binarized). If one need something really binarized on the page after that he can create a zone on it and set its properties to "Substract from all layers".  Or if binarization process missed some pieces of images that should be in the foreground - you may add a zone with "Add to the image layer" property.  

Personally I'm using Foreground layer to keep damaged or dirty (library stamps etc.) lines of text not binarized and left them readable.

**Reply by AmyYummy** [reply]
Try it with Illustrator, make vector words, no need for photo[shop](www.baudcom.cn).

**Comment by Cix75** [comment]
yes, I tried all the options and compared them in photoshop but the result is very similar, the same I would say.
it seems that the outlines of the letters are always too much sharp in the same way as B&W.  :(

**Reply by trufanov-nok** [reply]
The result pixels are the same as the original page contains (after the mask is applied). I guess *too much sharp* means bw mask threshold used to separate white from content isn't sensitive enough so it looses some border grayscale pixels. If your page contains only a foreground layer you can just increase bw threshold. If it contains both foreground and bw layer there is an option in STU settings in Output\Mixed which will display a separate threshold settings in GUI for bw and foreground detection. You may play with the least one.   

Also app settings contains an option to disable the bw layer smoothing. ST uses own smoothing after binarization. Some users hate it so there is such option. You may try it. I would suggest to choose regenerate command after its change from page context menu to see the difference in page rendering. And if it's better when it's off - delete all result files in the output  to regenerate the project (STU can't automatically detect this option change yet, bcs it's rarely used).
